src/theme/ItaliaTheme/Subsites/ItaliaTheme/Components/_subsiteFooter.scss

```diff
@@ -9,16 +15,40 @@
       .text {
         color: $subsite-primary-text;
 
+        .draftjs-buttons a {
+          @if $subsite-primary != $subsite-secondary {
+            background-color: $subsite-secondary !important;
+            color: $subsite-secondary-text !important;
+            text-decoration: none;
+
+            &:hover {
+              background-color: darken($subsite-secondary, 5%) !important;
+            }
+          } @else {
+            background-color: $subsite-secondary-text !important;
+            color: $subsite-secondary !important;
+            text-decoration: none;
+
+            &:hover {
+              background-color: darken($subsite-secondary-text, 5%) !important;
+            }
+          }
+        }
+
         h1,
         h2,
         h3,
         h4,
         a {
           color: $subsite-primary-text;
+        }
+
+        a {
+          text-decoration: underline;
 
           &:hover,
           &:active {
-            color: $subsite-primary-text;
+            text-decoration: none;
           }
```

Comment:
Se questa parte non dipende dai parametri dei singoli sottositi, possiamo metterla una tantum in un file di stili che si applica a tutti i sottositi, invece che nei mixin? Così viene generato questo codice una volta sola e non una per ciascun sottosito